docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | Any Kargo resource | Arbitrary string (acts as a token) | Triggers reconciliation when the value changes. Useful for forcing a resync. |
```

Comment:
"Any Kargo resource" isn't really true. Off the top of my head, I think this works only on Warehouses, Stages, and Promotions.

This sounds like it's just saying the same thing twice:

> Triggers reconciliation when the value changes. Useful for forcing a resync.